scania-digital-design-system/sdds
# fix(stepper): updated the stepper stories, to include the correct property

**Describe pull-request**  
Updated the stepper stories, to include the correct property.


**Solving issue**  
Fixes: -

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Stepper
3. Make sure the story has one "current" step. 

**Suggested test steps**
